### DISCUSSION TITLE
RSS Podcast Site

### DISCUSSION THREAD

**Discussion by candidexmedia** [GetPublii/Publii#1402] [Ideas]
Hi all,

This is a continuation of old forum posts on using Publii's RSS generator for podcasting:

- [Recommendations for podcast](https://forum.getpublii.com/topic/recommendations-for-podcast/)
- [Can I set up RSS for audio?](https://forum.getpublii.com/topic/can-i-set-up-rss-for-audio/)
- (loosely) [Distinct RSS Feeds for Each Tag](https://forum.getpublii.com/topic/distinct-rss-feeds-for-each-tag/)

I have some hunches about how it could actually work, but I haven't tested any theories (yet!)

This is a space to share insights, ideas, experiments, etc.

Here are my initial findings on analytics, audio storage and streaming, audio players, RSS spec for podcasts, etc.: https://gist.github.com/candidexmedia/743c1128e272f155ecab5b4d4324e6ee

**Comment by PresGas** [comment]
I am commenting to +1 the [Distinct RSS Feeds for Each Tag](https://forum.getpublii.com/topic/distinct-rss-feeds-for-each-tag/). 

Overall more robust rss generation would be awesome! Perhaps I should make a separate post...

**Comment by rgblack316** [comment]
I am hosting my RSS feed for my podcasts on Publii sites. I just put the .xml feed file in the root directory in the files manager to upload. I just swap out the file when I need to update and sync the site. I generate my feed with Sovereign Feeds. It's a great online, free tool for building a podcast feed and supports most, if not all, of the Podcasting 2.0 features.

**Reply by candidexmedia** [reply]
That's interesting! I've never heard of Sovereign Feeds, but I'll give it a look. Do you enjoy using it, or would you prefer generating the feed from Publii's end?

Also, where do you host your audio files?

I've been slowly digging into the podcast feed format, post templates in Publii, and how the [Post Settings API](https://getpublii.com/dev/theme-settings-api/#howtocreatepostorpagesettings) could (potentially maybe) be used to create podcast RSS entries for each episode... I know it's possible to override the feed's template by putting the [feed's handlebars template](https://github.com/GetPublii/Publii/blob/master/app/default-files/theme-files/feed-xml.hbs) into the site's theme folder (and potentially [tweaking the feed helper](https://github.com/GetPublii/Publii/blob/master/app/back-end/modules/render-html/handlebars/helpers/feed-link.js)).
